slitlessutils/core/preprocess/crrej/group.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+
+from slitlessutils.core.wfss import WFSSCollection
+from ....logger import LOGGER
```

Comment:
```suggestion
from slitless_utils import LOGGER
```
I think this or `from slitlessutils.logger import LOGGER` should work and avoid all the dots.

Reply:
I think it should not have an underscore?  `from slitlessutils import LOGGER` ?

Reply:
Whoops, good catch - got it right in one spot and not the other.